src/libData/AccountData/AccountStoreSC.tpp

```diff
@@ -350,119 +517,125 @@ bool AccountStoreSC<MAP>::UpdateAccounts(const uint64_t& blockNum,
         if (ret) {
           std::string runnerPrint;
 
-          // invoke scilla runner
-          InvokeInterpreter(RUNNER_CREATE, runnerPrint, scilla_version,
-                            is_library, gasRemained, amount, ret, receipt);
-
-          // parse runner output
-          try {
-            if (ret && !ParseCreateContract(gasRemained, runnerPrint, receipt,
-                                            is_library)) {
-              ret = false;
-            }
-            if (!ret) {
-              gasRemained = std::min(
-                  transaction.GetGasLimit() - createGasPenalty, gasRemained);
+          if (isScilla)
+            InvokeInterpreter(RUNNER_CREATE, runnerPrint, scilla_version,
+                              is_library, gasRemained,
+                              std::numeric_limits<uint128_t>::max(), ret,
```

Comment:
Why has the amount replaced with std::numeric_limits<uint128_t>::max()? Seems dangerous.

Reply:
That's an error I bet I know who did that, looking inwards.